### PR TITLE
i18n(zh-tw): Update `docsearch.ts`, `nav.ts` and `ui.ts`

### PR DESCRIPTION
**Pull request by NightFeather0615**
#### What kind of changes does this PR include?
- Translated content

#### Description
* Update `zh-tw/docsearch.ts`
* Update `zh-tw/nav.ts`
* Update `zh-tw/ui.ts`

### CODE REVIEW

**Comment by ElianCodes** [comment]
Merging `main` in here to make sure it's rebased on v3

File: src/i18n/zh-tw/nav.ts [hunk 1]
```diff
@@ -5,50 +5,63 @@ export default NavDictionary({
 	'getting-started': '新手上路',
 	install: '安裝',
 	'editor-setup': '編輯器設定',
+	'guides/upgrade-to/v2': '升级到 v2',
```
Comment (Yan-Thomas) [inline src/i18n/zh-tw/nav.ts:8]: This should fix the CI issue!
```suggestion
	'guides/upgrade-to/v3': '升级到 v3',
```

**Review by Yan-Thomas — COMMENTED** [review]
Thanks for this @NightFeather0615, I added a suggestion to fix the CI issue, and I'll also let translators in our Discord know of this PR so they can review it!

**Review by Genteure — APPROVED** [review]
LGTM 👍

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @NightFeather0615, and welcome to Team Docs! 🥳

**Pull request merged**